app/views/documents/index.html.erb

```diff
@@ -18,7 +18,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <% if current_user.assessor? && (!@planning_application.determined? || !@planning_application.withdrawn? || !@planning_application.returned?) %>
+    <% if @planning_application.cancellable? && current_user.assessor? %>
```

Comment:
So from my reading, this commit doesn't actually change the functionality? planning_application.cancellable? is the same as what was there before. This is a much nicer refactor though.

Reply:
Yes, but it's totally weird... this works as expected and for some reason the previous iteration didn't seem to.

Reply:
Hmm, I think maybe we're getting confused with logical arguments. This is why we write tests :-D